.ebextensions/cloudwatch.config

```diff
@@ -11,12 +11,16 @@ sources:
   /opt/cloudwatch: https://aws-cloudwatch.s3.amazonaws.com/downloads/CloudWatchMonitoringScripts-1.2.1.zip
 
 container_commands:
-  01-setupcron:
+  01-setupmetricscron:
     command: |
       echo '*/5 * * * * root perl /opt/cloudwatch/aws-scripts-mon/mon-put-instance-data.pl `{"Fn::GetOptionSetting" : { "OptionName" : "CloudWatchMetrics", "DefaultValue" : "--mem-util" }}` >> /dev/null 2>&1' > /etc/cron.d/cwpump
-  02-changeperm:
+  02-setupmemoryusagemetricscron:
+      echo '*/5 * * * * aws cloudwatch put-metric-data --metric-name MemoryUsage --namespace DelayedJob --value=$(./current/script/delayed_job_memory_usage.sh) --unit Count --region=eu-west-2 --dimensions InstanceId=$(ec2-metadata --instance-id | cut -d " " -f 2)' > /etc/cron.d/cwpump
+  03-setupnumberofprocessesscron:
+      echo '*/5 * * * * aws cloudwatch put-metric-data --metric-name Processes --namespace DelayedJob --value=$(./current/script/delayed_job_number_of_processes.sh) --unit Count --region=eu-west-2 --dimensions InstanceId=$(ec2-metadata --instance-id | cut -d " " -f 2)' > /etc/cron.d/cwpump
```

Comment:
set for every 5 mins as above

Reply:
@ldodds i'm thinking we should remove these from this pr and set up manually to test.  what do you think?

Reply:
As discussed, remove this from this PR and create a separate PR with a change to the cronjob.config to create the necessary cron file.

Reply:
:+1: done @ldodds